.github/workflows/reusable_workflow.yaml

```diff
@@ -79,34 +76,60 @@ jobs:
           archetype: ${{ inputs.archetype }}
           req_check_only: ${{ inputs.multi_arch }}
 
-      - name: Set up Docker Buildx for multi-arch
-        if: ${{ inputs.multi_arch }}
-        uses: docker/setup-buildx-action@v3
-
-      - name: Login to Quay for multi-arch
+      - name: Log in to quay.io for multi-arch
         if: ${{ inputs.multi_arch }}
-        uses: docker/login-action@v3
+        uses: redhat-actions/podman-login@v1
         with:
           registry: quay.io
           username: ${{ secrets.QUAY_USERNAME }}
           password: ${{ secrets.QUAY_PASSWORD }}
-
-      - name: Build and push multi-arch image for release
+      
+      - name: Build multi-arch image for release
+        id: build-multi-arch-release
         if: github.event_name == 'release' && inputs.multi_arch == true
-        uses: docker/build-push-action@v5
+        uses: redhat-actions/buildah-build@v2
         with:
           context: .
-          platforms: linux/amd64,linux/arm64
-          push: true
+          platforms: linux/amd64, linux/arm64
```

Comment:
I don't disagree, but I'm not sure how deep a change like that goes.

Reply:
This would need to be passed in with the workflow yaml of each repo that would be calling the reusable workflow. The same way the other inputs are being used for example image_name or image_tag. This would allow flexibility on which repos (If desired) build what type of archetype. I will say that when testing this months ago now, I was only able to build these two though. Any IBM archetypes failed complaining on dependencies that when included broke our original two builds. Not saying it's impossible just worth noting.